app/resources/ama_resource.rb

```diff
@@ -0,0 +1,10 @@
+class AMAResource < BaseResource
+  attributes :description, :start_date, :end_date,
+    :ama_subscribers_count
+
+  has_one :author
+  has_one :original_post
+  has_many :ama_subscribers
+
+  filters :end_date, :start_date, :ama_subscribers_count
```

Comment:
What do you mean? If you want past end_dates, can you do a less/greater than end_date query from the client with the current date?